### PR TITLE
[DOC-7786] Revamp of AWS Connect docs

### PR DESCRIPTION
**Pull request by urbiz-nr**
As explained in DOC-7786 and DOC-7787.
Also moving Cloud uninstall, as requested by PM.

### CODE REVIEW

**Comment by urbiz-nr** [comment]
Reviewer! Please check the deploy preview to see how the new section and docs look. I haven't changed too much content, it's mostly structure and info org. I've looked at it and haven't found anything odd. Thanks!

**Review by barbnewrelic — COMMENTED** [review]
Our style guide has some guidelines about how to refer to Amazon products. At first glance this does not seem to match what https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#aws recommends. But I may be over thinking this, so I trust your judgment.

File: src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx [hunk 1]
```diff
@@ -9,10 +9,26 @@ redirects:
...
+    * `AWS/ApplicationELB -> aws.applicationelb`
+  * The original AWS metric name with its original case:
+    * `aws.ec2.CPUUtilization`
+    * `aws.s3.5xxErrors`
+    * `aws.sns.NumberOfMessagesPublished`
+  * If the resource the metric belongs to has a specific namespace prefix, it is used. If the resource the metric belongs to doesn't have a specific namespace prefix, metrics use the `aws.` prefix.
+    * `aws.Region`
+    * `aws.s3.BucketName`
+
+Current namespaces supported by AWS can be found in the [CloudWatch documentation website](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html).
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx:28]: Rather than passive voice here, maybe change to "For more information about namespaces supported by AWS, see  the [Amazon AWS CloudWatch documentation](link)."

File: src/content/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase.mdx [hunk 2]
```diff
@@ -48,12 +48,12 @@ After setting up New Relic Infrastructure Amazon integrations, your usage of the
...
     title="Review API usage"
   >
-    To review the API usage for New Relic Infrastructure integrations with Amazon AWS:
+    To review the API usage for New Relic Infrastructure integrations with AWS:
 
     1. Go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > AWS > Account status dashboard**.
     2. Review the dashboard.
 
-    The dashboard includes a chart with your account's Amazon AWS API call count for the last month as well as the CloudWatch API calls (per AWS resource) for the last day. This information is the API usage for New Relic only. It does not include other AWS API or CloudWatch usage that may occur.
+    The dashboard includes a chart with your account's  API call count for the last month as well as the CloudWatch API calls (per AWS resource) for the last day. This information is the API usage for New Relic only. It does not include other AWS API or CloudWatch usage that may occur.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase.mdx:56]: looks like extra space between "account's" and "API"

File: src/content/docs/infrastructure/google-cloud-platform-integrations/get-started/connect-google-cloud-platform-services-new-relic.mdx [hunk 3]
```diff
@@ -171,49 +171,70 @@ You can [disable any of your GCP integrations](/docs/infrastructure/new-relic-in
...
-
+    
     <tr>
       <td>
         Clean your GCP Projects after unlinking New Relic
       </td>
 
       <td>
-        To clean your GCP project after unlinking, follow these steps if you were using a service account:
+        To clean your GCP project after unlinking, follow the steps below:
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/google-cloud-platform-integrations/get-started/connect-google-cloud-platform-services-new-relic.mdx:223]: "follow these steps"?

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx [hunk 4]
```diff
@@ -105,6 +105,8 @@ To add your Azure app to New Relic:
 2. Follow the steps in the UI to activate the integration in New Relic.
 3. If you have already completed the Azure account steps, skip to the end of the steps to fill out the form. (For `Azure account name`, enter the name you want to use to identify the account in your **Integrations** dashboard.)
 
+After you activate an Azure integration, we'll start monitoring your Azure data at regular [polling intervals](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-polling-intervals-infrastructure-integrations). To [find and use your data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data), use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) or go to [**one.newrelic.com**](https://one.newrelic.com) **> Infrastructure > Azure**.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx:108]: Looks like there is extra ** in the path info

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx [hunk 5]
```diff
@@ -114,6 +116,53 @@ Follow these steps to rotate the Azure client secret in the Infratructure UI:
 2. Select the edit action next to Account Name to see and edit any application value.
 3. Edit the **Client Secret** field with the new value and confirm with **Save Changes**.
 
-## Explore app data in New Relic Infrastructure's UI [#find-data]
-
-After you activate an Azure integration, New Relic will start monitoring your Azure data at regular [polling intervals](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-polling-intervals-infrastructure-integrations). To [find and use your data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data), use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) or go to [**one.newrelic.com**](https://one.newrelic.com) **> Infrastructure > Azure**
+## Uninstall Azure integrations [#uninstall]
+
+To uninstall Azure integrations, see the table below.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx:121]: "follow these steps" instead of "see the table below"?
This is just me. I don't like to refer to "above" or "below" in a doc because you never know when the info might get moved! :-O

File: src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 6]
```diff
@@ -17,15 +17,16 @@ redirects:
...
 
-With the New Relic Explorer, we give you a new, efficient front-door experience to easily observe the full stack of your software. We automatically create and maintain visibility from the data you send us. Use the New Relic Explorer to access and give context to the performance data from all your monitored applications, services, hosts, containers, synthetic monitors, etc. You can quickly see the entities related to a problem, exposing possible root causes and what other systems might be affected.
+The New Relic Explorer it's our new, efficient front-door experience to easily observe the full stack of your software. We automatically create and maintain visibility from the data you send us. Use the New Relic Explorer to access and give context to the performance data from all your monitored applications, services, hosts, containers, synthetic monitors, etc. You can quickly see the entities related to a problem, exposing possible root causes and what other systems might be affected. Want to try it? [Create a New Relic account](https://newrelic.com/signup) for free, forever!
```
Comment (barbnewrelic) [inline src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:25]: Explorer is no longer new, plus there's a missing verb. Maybe reword to "The New Relic Explorer gives you an efficient, front-door experience" etc

File: src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 7]
```diff
@@ -123,28 +124,32 @@ Read more about [New Relic Lookout](/docs/new-relic-one/use-new-relic-one/core-c
...
 
-![New Relic One entity alert status](./images/new-relic-one-entity-alert-status-red.png "new-relic-one-entity-alert-status.png")
+<ImageSizing width="320px" verticalAlign="middle">![New Relic One entity alert status](./images/new-relic-one-entity-alert-status-red.png "new-relic-one-entity-alert-status.png")</ImageSizing>
 
 * To see what an alert status means, mouse over it.
 * To see details about an entity's alerting status, select the entity.
 
 [NRQL alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries) aren't used to determine alert status because they aren't associated with specific entities.
 
-<Callout variant="important">
```
Comment (barbnewrelic) [inline src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:133]: Thanks for deleting this old callout!

**Review by barbnewrelic — DISMISSED** [review]
Wow, this was a huge effort and looks really good in Gatsby preview! I made some very minor questions and comments in my review, but overall I am impressed with what you have accomplished. Congrats!

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+  </tbody>
+</table>
+
+Want to try out our Amazon CloudWatch Metric Streams integration? [Sign up with New Relic](https://newrelic.com/signup) for free, forever! 
+
+### Cost considerations [#cost-considerations]
+
+Consider the following when evaluating the cost of the AWS CloudWatch metric streams integration with New Relic:
+
+* [AWS CloudWatch metric updates](https://aws.amazon.com/cloudwatch/pricing/). See **Metric Streams**.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:76]: Good catch!

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+Alternatively, you can use filtering on queries to distinguish between metrics that come from Metric Streams and those that come through polling. All metrics coming from Metric Streams are tagged with `collector.name='cloudwatch-metric-streams'`.
+
+### Migration steps [#migration-steps]
+
+On a typical deployment, migrating from API polling to metric stream involves the following steps (we recommend trying this on a dev / staging environment first):
+
+1. Go through the AWS UI in New Relic (or use [NerdGraph APIs](/docs/apis/nerdgraph/examples/nerdgraph-cloud-integrations-api-tutorial/#link-aws-cloudwatch)) to link your AWS account with New Relic. This is currently needed even if your AWS account is already linked with polling integrations.
+2. Make sure you complete the last step in the onboarding, which involves enabling AWS CloudWatch metric stream and the AWS Kinesis Data Firehose to push metrics to New Relic.
+  - Complete this step for any additional AWS region you want to monitor, since AWS CloudWatch requires one stream per region.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:95]: Good idea!

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+### Migration steps [#migration-steps]
+
+On a typical deployment, migrating from API polling to metric stream involves the following steps (we recommend trying this on a dev / staging environment first):
+
+1. Go through the AWS UI in New Relic (or use [NerdGraph APIs](/docs/apis/nerdgraph/examples/nerdgraph-cloud-integrations-api-tutorial/#link-aws-cloudwatch)) to link your AWS account with New Relic. This is currently needed even if your AWS account is already linked with polling integrations.
+2. Make sure you complete the last step in the onboarding, which involves enabling AWS CloudWatch metric stream and the AWS Kinesis Data Firehose to push metrics to New Relic.
+  - Complete this step for any additional AWS region you want to monitor, since AWS CloudWatch requires one stream per region. 
+3. Ensure metrics are received from all connected regions and namespaces. This may take several minutes.
+4. Disable all unnecessary polling integrations in the previous AWS provider account. The following integrations still need to be enabled since they aren't fully replaced by metric streams:
+  - AWS Billing, AWS CloudTrail, AWS Health, AWS Trusted Advisor.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:98]: I've reworded this and added a link to a more relevant section.

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+
+### Query, dashboard, and alert considerations [#considerations]
+
+AWS Metric Streams integration uses the Metric API to push metrics in the dimensional metric format.
+
+Poll-based integrations push metrics based on events (for example, `ComputeSample` event), and will be migrated to dimensional metrics in the future.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:104]: It's a perpetual WIP lol!

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+
+To assist in this transition, New Relic provides a mechanism (known as shimming) that transparently lets you write queries in any format. Then these queries are processed as expected based on the source that's available (metrics or events). This mechanism works both ways, from events to metrics, and viceversa.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:106]: Thanks!

File: src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+tags:
+  - Infrastructure
+  - Install AWS
+  - Update or uninstall
+metaDescription: How to uninstall AWS integrations.
+---
+
+[Uninstalling the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/installation/uninstall-infrastructure-agent-or-integrations) doesn't automatically uninstall any of your [infrastructure integrations](/docs/infrastructure/infrastructure-integrations/getting-started/connect-aws-integrations-infrastructure).
+
+To uninstall [AWS Metric Stream](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) or any other AWS integration, see the table below.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx:12]: Got it!

File: src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+      <tbody>
+        <tr>
+          <td id="disable-aws-integrations">
+            Disable AWS CloudWatch metric streams integration
+          </td>
+
+          <td>
+            To disable AWS CloudWatch metric streams integration:
+
+            1. From AWS Console, remove all required resources that were created during AWS integration setup. These include: AWS CloudWatch stream, AWS Kinesis Data Firehose endpoint and the AWS S3 backup bucket. If you used the AWS CloudFormation template, then follow the steps to [delete a stack on the AWS CloudFormation console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-delete-stack.html).
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx:36]: Thanks!

File: src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+          <td id="disable-aws-integrations">
+            Disable AWS CloudWatch metric streams integration
+          </td>
+
+          <td>
+            To disable AWS CloudWatch metric streams integration:
+
+            1. From AWS Console, remove all required resources that were created during AWS integration setup. These include: AWS CloudWatch stream, AWS Kinesis Data Firehose endpoint and the AWS S3 backup bucket. If you used the AWS CloudFormation template, then follow the steps to [delete a stack on the AWS CloudFormation console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-delete-stack.html). 
+            2. Complete step 1 for each active AWS region where you connected the AWS CloudWatch metric stream. 
+            3. From **AWS Status Dashboard** in Infrastructure > AWS, make sure no new metric updates are being ingested in New Relic.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx:38]: I like it!

File: src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+            4. Complete the steps to [Disconnect the AWS account](#unlink-aws).
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/uninstall-aws.mdx:39]: Good catch!

File: src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx:28]: Good one! Thanks!

File: src/content/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase.mdx (hunk 2 above)
Comment (urbiz-nr) [reply src/content/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase.mdx:56]: there was! you rock!

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx (hunk 4 above)
Comment (urbiz-nr) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx:108]: Correct!

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx (hunk 5 above)
Comment (urbiz-nr) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx:121]: Thanks!

File: src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx (hunk 7 above)
Comment (urbiz-nr) [reply src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:133]: 👍

File: src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx (hunk 6 above)
Comment (urbiz-nr) [reply src/i18n/content/jp/docs/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:25]: Yep!
